llmfoundry/utils/builders.py

```diff
@@ -190,9 +190,11 @@ def build_tokenizer(
 
     signal_file_path = f'.node_{dist.get_node_rank()}_local_rank0_completed_tokenizer_setup'
 
-    # Make sure the tokenizer files are downloaded and cached first by local rank 0
-    with dist.local_rank_zero_download_and_wait(signal_file_path):
-        pass
+    if dist.is_available() and dist.is_initialized(
```

Comment:
Do you need both is_available and is_initialized?

Reply:
thats what we use everywhere else. Do I not need both?

Reply:
Lets just leave both